charts/shield/templates/cluster/clusterrole.yaml

```diff
@@ -121,6 +121,24 @@ rules:
   verbs:
     - get
 {{- end }}
+{{- if .Values.cluster_shield.features.kubernetes_metadata.enabled }}
+- apiGroups:
+    - coordination.k8s.io
+  resources:
+    - leases
+  verbs:
+    - get
+    - watch
+    - list
+- apiGroups:
+    - ""
+  resources:
+    - secrets
+  verbs:
+    - get
+    - watch
+    - list
+{{- end }}
```

Comment:
U can remove the `if`statement and add the 2 blocks in the one below

Reply:
Done https://github.com/sysdiglabs/charts/compare/8171c2c3467b9fcbad890091f6fcfe2664194a57..b5b6efcf0c974f6b77d457018edc1b46f1663a92